src/main/java/pl/ug/virtualofficebackend/domain/entity/RoomEntity.java

```diff
@@ -0,0 +1,109 @@
+package pl.ug.virtualofficebackend.domain.entity;
+
+import javax.persistence.*;
+import java.util.Collection;
+import java.util.Objects;
+
+@Entity
+@Table(name = "room", schema = "public", catalog = "postgres")
+public class RoomEntity {
+    private int roomId;
+    private int typeId;
+    private String name;
+    private int capacity;
+    private Collection<OfficeLayoutEntity> officeLayoutsByRoomId;
+    private RoomTypeEntity roomTypeByTypeId;
+    private Collection<RoomLayoutEntity> roomLayoutsByRoomId;
+
+    @Id
+    @Column(name = "room_id")
+    public int getRoomId() {
+        return roomId;
+    }
+
+    public void setRoomId(int roomId) {
+        this.roomId = roomId;
+    }
+
+    @Basic
+    @Column(name = "type_id")
+    public int getTypeId() {
+        return typeId;
+    }
+
+    public void setTypeId(int typeId) {
+        this.typeId = typeId;
+    }
+
+    @Basic
+    @Column(name = "name")
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Basic
+    @Column(name = "capacity")
+    public int getCapacity() {
+        return capacity;
+    }
+
+    public void setCapacity(int capacity) {
+        this.capacity = capacity;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        RoomEntity that = (RoomEntity) o;
+
+        if (roomId != that.roomId) return false;
+        if (typeId != that.typeId) return false;
+        if (capacity != that.capacity) return false;
+        if (!Objects.equals(name, that.name)) return false;
+
+        return true;
+    }
+
+    @Override
+    public int hashCode() {
+        int result = roomId;
+        result = 31 * result + typeId;

```

Comment:
kod sam Ci to generował? nie wiem, czy nie powinno brać po kolei liczb pierwszych, czy to, że są takie same, to nie jest problem

Reply:
To jest kod wygenerowany przez ide.
ten hashCode jest imo useless na ten moment.